scripts/test_env.sh

```diff
@@ -97,11 +98,14 @@ function run_peer () {
     STORAGE="$PEER/storage"
     mkdir -p "$STORAGE"
     KURA_BLOCK_STORE_PATH="$STORAGE"
+	LOG_FILE_PATH="$PEER/log.json"
+	MAX_LOG_LEVEL="TRACE"
     TORII_P2P_ADDR="$HOST:${p2p_ports[$1]}"
     TORII_API_URL="$HOST:${api_ports[$1]}"
     TORII_TELEMETRY_URL="$HOST:${telemetry_ports[$1]}"
     IROHA_PUBLIC_KEY="${public_keys[$1]}"
     IROHA_PRIVATE_KEY="${private_keys[$1]}"
+	SUMERAGI_DEBUG_FORCE_SOFT_FORK="false"
```

Comment:
formatting

Reply:
for other above as well

Reply:
you seem to have a problem with bash scripts

Reply:
New work machine. Emacs by default uses tabs for shell scripts and C. I haven't had the time to fully migrate my old config to M1.